tools/StaticAnalysis/Exceptions/Az.Billing/BreakingChangeIssues.csv

```diff
@@ -1,4 +1,4 @@
-"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"

```

Comment:
what is the change here? if there is no change, we can skip this file right?

Reply:
it was part of a merge resolution.